.vscode/launch.json

```diff
@@ -5,20 +5,11 @@
             "name": "Python Module",
             "type": "python",
             "request": "launch",
-            "stopOnEntry": true,
             "module": "iotedgedev.cli",
             "args": [
-                "solution",
-                "new",
-                "test_solution"
+                "push"
```

Comment:
this change is by design?

Reply:
Yes. The launch.json serves as the template for this project' contributor. Just found that the current one contains some now-invalid property such as `debugOptions`, so I rewrote it.